docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3075.md

```diff
@@ -0,0 +1,29 @@
+# Audit pipe mount operations
+
+Test verifies that Pipe mount read, write delete operations are monitored and logged.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- perform [3059_2](3059_2.md) case
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the non-admin user from the prerequisites | |
+| 2 | At the Runs page, click the run launched at step 4 of [3059_1](3059_1.md) case | |
+| 3 | Wait until the SSH hyperlink appears | |
+| 4 | Click the SSH hyperlink | |
+| 5 | In the opened tab, enter and perform the `echo "test info" >> cloud-data/storage2/file3` command | |
+| 6 | Logout and login as the admin user from the prerequisites | |
+| 7 | Open the **Settings** page | |
+| 8 | Open the **SYSTEM MANAGEMENT** page | |
+| 9 | Select the **LOGS** tab | |
+| 10 | Click ***Show advanced*** button | |
+| 11 | Click **Type** field and select `audit` value from drop-down | Record <br> `WRITE storage2/file3` <br> is shown in the top of logs records |
```

Comment:
I suggest to add an action like "Clear the field **From**" - as by default logs are shown from the moment of the opening **Logs** form.